### PR TITLE
Fix Pack using MsBuild 15

### PR DESCRIPTION
**Pull request by toddm**
Fixes https://github.com/NuGet/Home/issues/2356

@emgarten @spadapet

### CODE REVIEW

**Comment by emgarten** [comment]
Looks way better :rocket:

**Comment by toddm** [comment]
Thanks.  I'll push it to Dev.

File: src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs [hunk 1]
```diff
@@ -362,58 +364,21 @@ private void BuildProject()
...
-                    throw new CommandLineException(LocalizedResourceManager.GetString("FailedToBuildProject"), Path.GetFileName(_project.FullPath));
-                }
+            string properties = string.Empty;
+            foreach (var property in ProjectProperties)
+            {
+                properties += $" /p:{property.Key}={property.Value}";
+            }
 
-                TargetPath = ResolveTargetPath(result);
+            int result = MsBuildUtility.Build(_msbuildDirectory, $"{_project.FullPath} {properties} /toolsversion:{_project.ToolsVersion}");
```
Comment (spadapet) [inline src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs:373]: Do you need quotes around the full path in case it contains spaces?

Comment (toddm) [inline src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs:373]: Yes, I caught that after and fixed it.  Thanks!

**Pull request closed without merging**